helpers/settings.go

```diff
@@ -119,6 +121,9 @@ func (s *Settings) InitSettings(envVars EnvVars, env *cfenv.App) error {
 	if len(envVars.SMTPHost) == 0 {
 		return errors.New("Unable to find '" + SMTPHostEnvVar + "' in environment. Exiting.\n")
 	}
+	if len(envVars.TICSecret) == 0 {
```

Comment:
if we are making TIC secret mandatory, we need to find a way to replicate it locally. but i don't think we can due to our setup vs PCFDev.
Also, if we make it mandatory, we need to add it here too https://github.com/18F/cg-dashboard/blob/master/docker-compose.yml#L31

Whether or not it's mandatory could also be resolved by answering this comment: https://github.com/18F/cg-dashboard/pull/1219/files#r137849061